www/src/components/ReactPlayground.js

```diff
@@ -67,15 +67,6 @@ const StyledExample = styled('div')`
   }
 
   :global {
-    .badge + .badge,
```

Comment:
This ends up being more trouble than it's worth. People get confused at this being just a quirk of our docs page.